Services/CO.CDP.EntityVerification/Events/Values.cs

```diff
@@ -2,6 +2,8 @@ namespace CO.CDP.EntityVerification.Events;
 
 public record Identifier
 {
+    public const string PponSchemeName = "CDP-PPON";
```

Comment:
This value is used with Events. If an event is serialised, would this constant be also serialised? 

If the answer is yes, then it shouldn't be here :)

Reply:
Good point. Should we define the possible schemes in a JSON file and load it in?

Reply:
There's a task for this in the backlog :)

Reply:
For now, maybe let's define a dedicated class with constants? i.e `IdentifierSchemes.Ppon`

Reply:
Done.